src/components/gltf-model-plus.js

```diff
@@ -256,8 +257,12 @@ async function loadGLTF(src, contentType, preferredTechnique, onProgress) {
 
   gltf.scene.traverse(object => {
     if (object.material && object.material.type === "MeshStandardMaterial") {
-      object.material.envMap = envMap;
-      object.material.needsUpdate = true;
+      if (preferredTechnique === "KHR_materials_unlit") {
```

Comment:
see this wasn't so hacky! :)

(thanks for the PR when you are technically on vacation!)